consensus/istanbul/backend/backend.go

```diff
@@ -57,6 +58,16 @@ var (
 	errInvalidSigningFn = errors.New("invalid signing function for istanbul messages")
 )
 
+type AuthorizeInfo struct {
```

Comment:
If you look at what this does, I think Wallet is an appropriate name. It holds the adderess and provides functions to sign and decrypt.

Reply:
I like this.

TBH I'm not sure either about the relationship keys and signing functions :) I know that signHash relates to the Wallet.SignHash function (that was removed in upstream but kept here), so I'm guessing it's the ECDSA but...

Reply:
Hehe :), yeah, I'm not sure, I'd have to dig into the code to find out, but that's why it would be nice for people if it was just explicit.

Reply:
What about changing the name to Wallet? I think its more descriptive than AuthorizeInfo, and then `auth()` would become `wallet()`

Reply:
I thought about it, but this is a grouping of different properties from some of the validator wallets, and I would like to avoid the confusion when someone sees `backend.wallet...`. I also don't like `AuthorizeInfo`, but it feels like `wallet` would make the code a little bit more confusing.

Reply:
I realise now I was actually previously suggesting the name Wallets, would you consider that any better?

The reason I don't like AuthorizeInfo as a name is that the meaning of Authorize is:

>to give official permission for something to happen, or to give someone official permission to do something

So autrhorizing is an action, so it makes sense for the method to be called authorize, but what is being given in this authorize action is a set of keys and addresses, which in the blockchain space is often referred to as a wallet.

Its not a biggy, just interested to hear your thoughts on this?

Reply:
Well I hate AuthorizeInfo, so I'm all in for changing it. I think Wallets is acceptable. I'll change it.

Reply:
So one little last thing, now that its called Wallets and we have wallets() `ai` as a variable name doesn't make much sense. I'd be happy with `w` or `wal`.

Reply:
agh, find & replace betrayed me